client/src/components/MainNavbar/MainNavbar.jsx

```diff
@@ -1,47 +1,55 @@
 import React from 'react'
+import { authActions } from '../../actions/auth'
 import { NavLink } from 'react-router-dom'
 import { withRouter } from 'react-router'
 import { connect } from 'react-redux'
-import GoogleAuth from '../GoogleAuth/GoogleAuth'
 import api from '../../apis/backend'
 class MainNavbar extends React.Component {
-  handleLogoutClick = () => {
-    api.logout()
-  }
-
   render() {
     return (
       <nav className="App-header">
         <div className="ui secondary pointing menu">
-          <NavLink className="item" to="/" exact>
-            Home
-          </NavLink>
-          <NavLink className="item" to="/games">
-            Games
-          </NavLink>
+          <div className="ui simple dropdown">
+            <div className="item">
+              <i className="user icon" />
+              Account
+            </div>
+            <div className="menu">
+
+              {api.isLoggedIn() && (
+                <div className="item">
+                  <NavLink className="item" to="/" onClick={this.props.logout}>Logout</NavLink>
+                </div>)}
+
+              {api.isLoggedIn() && (
+                <div className="item">
+                  <NavLink className="item" to="/profile">My Profile</NavLink>
+                </div>
+              )}
```

Comment:
I think we can combine these conditional checks since they're both checking for the same thing.